.github/workflows/update-syscalls.yml

```diff
@@ -0,0 +1,36 @@
+name: Update Syscalls
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 1 * *'
+
+jobs:
+  update-syscalls:
+    runs-on: ubuntu-latest
+    container:
+      image: falcosecurity/syscalls-bumper:latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          
+      - name: Bump syscalls
+        run: |
+          syscalls-bumper --repo-root $(pwd) --overwrite
+          
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          signoff: true
+          base: syscalls
+          branch: update/syscalls
+          title: 'update(driver): update syscalls tables and driver report.'
+          body: 'This PR updates the list of supported syscalls from the latest kernel. Do not edit this PR.\n/area driver\n\`\`\`\release-note\nNONE\n\`\`\`'
+          commit-message: 'update(driver): update syscalls tables and driver report.'
+          user: poiana
+          token: ${{ secrets.POIANA_TOKEN }}
```

Comment:
We:
* need to install `git` package in the docker container
* can try to use `secrets.GITHUB_TOKEN` here instead
* `base` branch should be master (and can be removed, since it is the default, given that the action runs on a master workflow)